securedrop-proxy/debian/changelog

```diff
@@ -1,3 +1,10 @@
+securedrop-proxy (0.1.3) UNRELEASED; urgency=medium
```

Comment:
UNRELEASED here is a divergence from previous releases, though should not have any impact on apt. Might be worth revising this for the beta

Reply:
Yup, will update it in the next push.